src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs

```diff
@@ -84,6 +85,15 @@ public void TracestateValueIsInvalid(ReadOnlySpan<char> value)
             }
         }
 
+        [NonEvent]
+        public void MetricControllerException(Exception ex)
```

Comment:
why the name of the method this thing calls does not match? `MetricObserverException` vs `MetricControllerException`?

Reply:
my bad. fixed